src/test/java/org/mockbukkit/mockbukkit/plugin/PluginManagerMockTest.java

```diff
@@ -427,4 +427,17 @@ void test_privateConstructorPlugin()
 		assertEquals("No publicly available constructor for PrivateConstructorPluginProxy without parameters", exception.getCause().getMessage());
 	}
 
+	@Test
+	void loadPlugin_WithPaperPluginYml_PluginLoaded()
+	{
+		Plugin loadedPlugin = pluginManager.loadPlugin(PaperTestPlugin.class);
+		assertInstanceOf(JavaPlugin.class, loadedPlugin);
```

Comment:
Just to be sure that the correct class was loaded
```suggestion
		assertInstanceOf(PaperTestPlugin.class, loadedPlugin);
```